woocommerce/woocommerce
# Add a workaround for compatibility of TS5 with Storybook 6

### Changes proposed in this Pull Request:

We experienced the exact same issue as in Gutenberg: https://github.com/WordPress/gutenberg/pull/53445 where Storybook 6 has an incompatibility with TS5.

This workaround solves the problem. Until we can upgrade to Storybook 7 we'll need to keep this workaround.

### How to test the changes in this Pull Request:


1. Checkout this branch
2. make sure a `pnpm install` has been done.
3. Navigate to `tools/storybook` and run `pnpm storybook`. it should not crash and you should be able to access a local storybook site when the task finishes.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
